zebra-rpc/qa/rpc-tests/addnode.py

```diff
@@ -16,7 +16,8 @@ def __init__(self):
         self.num_nodes = 3
 
     def setup_network(self, split=False):
-        self.nodes = start_nodes(self.num_nodes , self.options.tmpdir)
+        args = [False, False, False]
+        self.nodes = start_nodes(self.num_nodes , self.options.tmpdir, extra_args=args)
```

Comment:
Optional: It may be a good idea to add another parameter for config modifications instead of re-using the extra_args param.

Subjective/Nitpick: This looks unnecessary (in a couple more cases below too).

```suggestion
        self.nodes = start_nodes(self.num_nodes , self.options.tmpdir)
```

Reply:
Let's move this here https://github.com/ZcashFoundation/zebra/issues/9720